spec/lib/pdf_filler/nj_additional_dependents_pdf_spec.rb

```diff
@@ -59,16 +59,6 @@
       expect(pdf_fields["SSN_Row6"]).to eq "300000068"
       expect(pdf_fields["BirthYear_Row6"]).to eq "2015"
       expect(pdf_fields["HealthInsurance_Row6"]).to eq "Off"
-
```

Comment:
Max 10 dependents in the XML means there will never be more than 6 dependents on the form.

Reply:
[dust] testing the absence of anything be filled in those rows could still be a tiny bit helpful, but no strong opinion